apps/design-system-docs/docs/components/packages/wp-components/status-indicator.tsx

```diff
@@ -0,0 +1,16 @@
+import { Tooltip } from '@wordpress/components';
+import { ComponentStatus, statuses } from './data';
+import styles from './status-indicator.module.scss';
+
+export const StatusIndicator = ( { status }: { status: ComponentStatus } ) => {
+	return (
+		<Tooltip text={ statuses.find( ( s ) => s.value === status )?.label }>
+			<div
+				className={ styles[ 'status-indicator' ] }
+				aria-label={ statuses.find( ( s ) => s.value === status )?.label }
```

Comment:
Is it necessary to have both `text=` and `aria-label=` ? Is the Tooltip not already handling the accessible name / description?

Reply:
Great question. Short answer is yes, we need both here because Tooltip only applies an accessible description on focus, and screen readers can read this icon without it being focused.

The long answer is more interesting. At some point we discovered that a tooltip probably shouldn't be adding an accessible string by default. That was reported up to Ariakit, and now Ariakit doesn't do that anymore (https://github.com/ariakit/ariakit/issues/3242). We haven't been able to propagate those new defaults to Gutenberg itself yet though, due to back compat (https://github.com/WordPress/gutenberg/pull/66021).